### ISSUE TITLE
Drop the setup

### ISSUE THREAD

**Issue by goktugyil** [bahlo/SwiftGif#10]
```swift
// jeremy.gif
var url = NSBundle.mainBundle().URLForResource("jeremy", withExtension: "gif")
var imageData = NSData(contentsOfURL: url)

// Returns an animated UIImage
UIImage.animatedImageWithData(imageData)
```

This uses too many lines to setup the gif. It could have been only:

```swift
 UIImage.withGifName("jeremy")
```

I might do this in the future when I have time

**Comment by goktugyil** [comment]
i can do this now

**Comment by bahlo** [comment]
@goktugyil That would be great! Looking forward to your PR :smirk:

**Comment by goktugyil** [comment]
https://github.com/bahlo/SwiftGif/pull/14

**Comment by bahlo** [comment]
Merged in [`1.5`](https://github.com/bahlo/SwiftGif/releases/tag/v1.5).
